### DISCUSSION TITLE
`Next` and `Previous` button does not get the translated title from `_category_.json`

### DISCUSSION THREAD

**Discussion by ygini** [facebook/docusaurus#9919] [Q&A]
Hello

We are working on our product documentation with Docusaurus and having and issue with the translated category.

As you can see on our left menu, the currently displayed page is between a partent category and a child one.

<img width="295" alt="image" src="https://github.com/facebook/docusaurus/assets/202334/fa6ada7f-d218-4cfc-94b3-f33f9d226ae4">

When reading the page up to the end, we see the previous and next button directed to those categories, but the label does not get translated.

<img width="1089" alt="image" src="https://github.com/facebook/docusaurus/assets/202334/833a2b84-ca72-4a45-985e-216ab0d9f696">

This is for all previous and next button each time they target a category instead of a page.

Is there a way to fix that?

**Comment by slorber** [comment]
This looks like this bug: https://github.com/facebook/docusaurus/issues/8118

**Reply by ygini** [reply]
Indeed, but about 2 years without accepting the fix?

**Reply by slorber** [reply]
No need to be condescending, remember that Docusaurus is free and you don't contribute to its sustainability.

The fix does not work, that's why it wasn't accepted.
If you have a fix that works, then feel free to open a PR

**Reply by ygini** [reply]
It’s not condescending, just short sentence and questioning. 

We don’t contribute so far indeed as we are still learning and evaluating the product to use it or not. 

And yes, I’m surprised that an issue that break the user experience and make the product not localizable is not fix. 

Which make me wonder if this project is still maintained or not.

**Reply by slorber** [reply]
Feel free to use another bug-free, well-maintained OSS software then 🤷‍♂️ 

Docusaurus has many other unfixed i18n bugs: https://github.com/facebook/docusaurus/issues/4542

**Reply by ygini** [reply]
Maybe a more useful contribution to this exchange on your side would be to explain why the proposed fix is not good enough (to the point that you prefer to keep a non working product) so we can see on our side if we can do the missing part. 

As usual with FOSS, if you explain clearly the situation and ask what you need, people will be more open to contribute.

**Reply by slorber** [reply]
It's difficult to answer this question without overheating the debate, but I'll try to briefly explain my position. 

- We all have limited time and need to figure out how to best spend it
- This issue is not critical, and not so easy to fix, even a maintainer couldn't fix it 2y ago
- There are more important bugs to fix and features to work on, with better ROI
- Answering questions, debating, and explaining things takes time, and if I'm not careful enough, I could spend 100% of my Docusaurus time doing that
- I don't have the problem in my head, and remembering things takes time
- You are asking me to give you some time, and I'm not super confident it will be time well spent

---

For that reason, I'm still open to helping you fix the problem, but I'm also going to ask you to do some homework first. 

Can you please convince me that investing time in explaining things is worth it?
- Have you read all the PRs and related issues?
- Have you read our docs, core plugin code, and understood how plugin lifecycles work?
- Have you studied the PR code, ran it, and understood why the bug happens and why the fix does not work?
- After doing that, do you have specific questions, feedback, or a different approach to suggest?

Most users simply won't do any of that, and maintainers need to protect their time accordingly.
